testcafe/tests/helpers/utils.js

```diff
@@ -29,6 +29,25 @@ export const getPageUrl = ClientFunction(() => window.location.href)
 
 export const goBack = ClientFunction(() => window.history.back())
 
+export const getNavigatorOs = ClientFunction(() => navigator.platform)
+
+//User Agent is needed for VisualReview, but we don't need the all string
+export const getNavigatorName = ClientFunction(() => {
+  let navigatorKey = ['MSIE', 'Firefox', 'Safari', 'Chrome', 'Opera'],
+    userAgent = navigator.userAgent,
+    index = navigatorKey.length - 1
+  for (
+    index;
+    index > -1 && userAgent.indexOf(navigatorKey[index]) === -1;
+    index--
+  );
+  return navigatorKey[index]
```

Comment:
you can just do:

`return navigatorKey.find(navigatorName => userAgent.indexOf(navigatorName) !== -1)`